### DISCUSSION TITLE
Accurate Thrust

### DISCUSSION THREAD

**Discussion by KingAsLan777** [flybywiresim/aircraft#5859] [Ideas]
I need a better accurate thrust, when I start moving the aircraft 20 knots, I set my throttle idle and the speed will not decrease it keeps on increasing I keep using the break and it starts decrease and increase doing fully stop the plane stop. It would be nice to make things a little bit realistic just like in the real world. it will help my landing performance nice and smooth.

**Comment by andrejetcat** [comment]
it is very realistic, that the plane can taxi with idle thrust. Even accelerate with no thrust is realistic. To my mind as a real world pilot, the thrust simulation is very realistic.
